src/content/docs/en/pages/main-menu/reference/build/edge-application/edge-caching.mdx

```diff
@@ -27,19 +23,48 @@ With Azion, you can customize the time the cache remains in the edge through *ti
 
 ---
 
+## Cache status
+
+Azion provides a layer to cache content in the edge network, which stands between origin servers and the end user. When content is successfully cached at the edge, it receives a HIT status and it can be directly delivered to your users from the edge node nearest to them, eliminating the frequency of access to origin servers.
+
+In the event of a MISS status, when the requested content isn't available in cache, the Edge Caching module minimizes the impact by maintaining a keep-alive connection with the origin server whenever possible. This approach avoids the overhead of frequent TCP/IP handshakes. Regardless of the number of simultaneous requests made to Azion's edge nodes, each edge node only needs to request content from the origin once per MISS.
+
+You can obtain the cache status of an application's resource by requesting the content URI and adding the Azion debug header `Pragma: azion-debug-cache`. A successful response will return an `X-Cache` header with the cache status, the IP of the edge node, and the protocol used in the request:
+
+```bash
+X-Cache: HIT from 200.000.000.00 with HTTP/2.0
+```
+
+:::note
+Find out more about how to obtain other cache data in the [How to verify application cache indicators using Modheader for Google Chrome](/en/documentation/products/guides/check-page-cache-time/) guide.
+:::
+
+### Status definitions
+
+| Status | Definition |
+| --- | --- |
+| **HIT** | Valid and up-to-date content is supplied directly from the edge cache. |
+| **MISS** | If the requested content isn't found in the cache at the edge, it's fetched from the origin. Through the response, the content might be cached for future requests. |
+| **EXPIRED** | The cached content has expired the TTL set at the edge. When the origin responds, the cached content is updated for future requests. |
+| **STALE** | When choosing to serve [stale cache](#stale-cache), if the origin fails to respond and the edge cache has expired, a stale version of the content is served. |
+| **UPDATING** | The cached content has expired and stale cache is being served, but the content will be updated for future requests. |
+| **BYPASS** | The edge requests the content from the origin directly instead of using a cached version due to the active [Bypass Cache](/en/documentation/products/edge-application/rules-engine/#bypass-cache) behavior. |
```

Comment:
```suggestion
| **BYPASS** | The edge requests the content from the origin directly instead of using a cached version due to the active [Bypass Cache](/en/documentation/products/edge-application/rules-engine/#bypass-cache) behavior. |
| **-** | If no status is received, the content requested is restricted from caching. For instance, if the requested content is a `POST` request and [Caching for POST](/en/documentation/products/edge-application/cache-settings/#caching-http-methods) is disabled, the status is not logged. |
```